synapse/app/_base.py

```diff
@@ -25,7 +25,9 @@
 from daemonize import Daemonize
 from synapse.util import PreserveLoggingContext
 from synapse.util.rlimit import change_resource_limit
-from twisted.internet import reactor
+from twisted.internet import error, reactor
+
+logger = logging.getLogger("synapse.app.client_reader")
```

Comment:
This is used outside the client reader. `logging.getLogger(__name__)`, please.

Reply:
Indeed. Fixed.